app/routes/notes/$noteId.tsx

```diff
@@ -46,18 +51,20 @@ export default function NoteDetailsPage() {
   );
 }
 
-export function ErrorBoundary({ error }: { error: Error }) {
-  console.error(error);
```

Comment:
We may want to keep this.  Render errors should get logged by our internal `componentDidCatch` but this would have also previously logged loader/action errors